### ISSUE TITLE
Add Prioritized Experience Replay for Rainbow implementation

### ISSUE THREAD

**Issue by Ashutosh-Adhikari** [hengyuan-hu/rainbow#1]
Should we work upon adding Prioritized Experience Replay ?

**Comment by hengyuan-hu** [comment]
yeah you are more than welcome to implement that. I had no access to GPU recently and just get one. But I personally may not give this high priority since a2c (https://github.com/hengyuan-hu/a2c) learns much faster.

**Comment by forhonourlx** [comment]
Why does a2c learn much faster? Should rainbow be the fastest overall?

**Comment by Ashutosh-Adhikari** [comment]
Hi @hengyuan-hu , you willing to merge the request on PER ? Or do you wish we should conduct more experiments ?
